examples/breakouttouchpaint/breakouttouchpaint.ino

```diff
@@ -15,11 +15,12 @@
 
 /** NOT FOR USE WITH THE TOUCH SHIELD, ONLY FOR THE 3.5" BREAKOUT! **/
 
-#include <Adafruit_GFX.h>    // Core graphics library
-#include <SPI.h>
+#include "Adafruit_mfGFX.h"
 #include "Adafruit_HX8357.h"
 #include "TouchScreen.h"
 
+SYSTEM_MODE(AUTOMATIC);
```

Comment:
We should probably do this in an #if SPARK to be compatible with the original arduino code the adafruit library was written for